paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -162,6 +207,40 @@ internal class CustomerSheetViewModel(
                 loadCustomerSheetState()
             }
         }
+
+        viewModelScope.launch {
+            selectPaymentMethodState.collectLatest { selectPaymentMethodState ->
+                updateViewState<CustomerSheetViewState.SelectPaymentMethod> {
+                    selectPaymentMethodState
+                }
```

Comment:
This is now the only usage of `updateViewState` that looks for `CustomerSheetViewState.SelectPaymentMethod`. Makes it much easier when updating to interactors.